### ISSUE TITLE
F2000 initial conditions

### ISSUE THREAD

**Issue by wwieder** [ESCOMP/CTSM#2558]
The CAM group, mainly @cecilehannay, is setting up an F2000 compset.  I assumed we'd use the same land configuration as in in the [091 run](https://github.com/NCAR/amwg_dev/issues/533) that Gustavo started with updates to datm.streams for the year 2000?  We also need to provide initial conditions.

Maybe we can use the initial conditions from the [2000pSASU](https://github.com/NCAR/LMWG_dev/issues/58) case that @slevis-lmwg ran recently?  Is this consistent with how we typically set up F2000 cases @olyson and @ekluzek?

**Comment by wwieder** [comment]
Let me know if there's a different place to have this discussion (e.g. AMGW-dev?). I'm relatively new to this cross working group coordination.

**Comment by PeterHjortLauritzen** [comment]
We are only supporting an F2000climo COMPSET for CESM3 (out-of-the-box).

**Comment by ekluzek** [comment]
So @PeterHjortLauritzen just to confirm that means you need 2000 finidat files -- but not 2010 then? Are the 2010 compsets/tests going to be removed in CAM?

This would be easier for us as we'd just support 2000 and remove 2010 compsets we had for CAM support.

**Comment by PeterHjortLauritzen** [comment]
There are some FC2010 compsets (full chemistry) so removal would need approval from the chemistry working group @tilmes

**Comment by slevis-lmwg** [comment]
Reminder: I still need to 
`./rimport -file lnd/clm2/initdata_esmf/ctsm5.2/clmi.I2000Clm52BgcCrop.2000-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c240515.nc`
I just tried it on casper (derecho is down) and got this error:
`ERROR: /glade/p/cesmdata/cseg/cseg/inputdata/lnd/clm2/initdata_esmf/ctsm5.2/clmi.I2000Clm52BgcCrop.2000-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c240515.nc does not exist`
I tried it on izumi and got this error:
`svn: E000028: Failed writing updated file: No space left on device`

**Comment by slevis-lmwg** [comment]
@ekluzek @wwieder 
I'm clarifying here based on info from today's meetings. Please comment/correct if I misunderstood.

First iteration of generating IC files occurs once the following work gets merged in ctsm master:
- Crop Calendar work
- Excess Ice
- Dust emissions
- Explicit A/C (just needs to be turned on by default)

Second iteration, after the following work gets merged in ctsm master:
- CTSM5.3 datasets
- MEGAN 2.1
- Fang's Fire PRs
- PPE branch

Third iteration, after gets in:
- Soil NOx work

**Comment by ekluzek** [comment]
Yes, we'll need to do a few different iterations of IC files.

And as such we should reduce how many we create until the final step when tuning has happened.

I think we might just need:

1850 spinup for f09 and ne30pg3?

So maybe you can skip doing 2000 IC files for now? @wwieder does that sound right?

**Comment by wwieder** [comment]
Maybe we make an F2000 ic file after step 2 that Sam mentioned above, when the 5.3 datasets come in? It really depends on how much other WGs want this, though so it's hard to say for sure.

**Comment by wwieder** [comment]
This seems like it may be a duplicate of #2403, at least regarding 5.3 datasets and initial conditions

**Comment by wwieder** [comment]
Closing this issue for now, as we we have ICs for 5.3 datasets and will be generating them again for the 5.4 release tag.
